main-app/app/scripts/bingoNumbers/checkingWinnerService.js

```diff
@@ -1,7 +1,7 @@
 (function () {
     'use strict';
     angular.module('Tombola.BingoClient.WinnerChecking')
-        .service('WinnerChecking', ['UserModel', function (UserModel) {
+        .service('WinnerChecking', ['UserModel', 'NumberService', function (UserModel, NumberService) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8112111/11587351/9784af24-9a70-11e5-97c8-f52e748c00dd.png)
The service name (the string) needs to be PascalCased, the variable names need to be camelCased.